backend/backend.go

```diff
@@ -717,7 +717,7 @@ func (backend *Backend) Start() <-chan interface{} {
 	backend.usbManager = usb.NewManager(
 		backend.arguments.MainDirectoryPath(),
 		backend.arguments.BitBox02DirectoryPath(),
-		backend.socksProxy,
+		*backend.socksProxy,
```

Comment:
It doesn't make much sense to make backend.socksProxy a pointer but then dereference it, or leave it as a value in the Manager struct :see_no_evil: